### PR TITLE
PEP-8 bodhi.server.util.

### PR DESCRIPTION
**Pull request by bowlofeggs**

### CODE REVIEW

File: bodhi/server/util.py [hunk 1]
```diff
@@ -16,47 +16,50 @@
...
-header = lambda x: u"%s\n     %s\n%s\n" % ('=' * 80, x, '=' * 80)
 
-pluralize = lambda val, name: val == 1 and name or "%ss" % name
+def header(x):
+    """Display a given message as a heading."""
+    return u"%s\n     %s\n%s\n" % ('=' * 80, x, '=' * 80)
+
+
+def pluralize(val, name):
+    return val == 1 and name or "%ss" % name
```
Comment (pypingou) [inline bodhi/server/util.py:65]: that got me confused a little, still kinda do, but somehow seems to work

(misses the docstring btw)

File: bodhi/server/util.py [hunk 2]
```diff
@@ -126,8 +129,6 @@ def get_age_in_days(date):
 
 def flash_log(msg):
     """ Flash and log a given message """
-    # FIXME: request.session.flash()
-    #flash(msg)
     log.debug(msg)
```
Comment (pypingou) [inline bodhi/server/util.py:135]: Should this method be entirely dropped then?

**Comment by pypingou** [comment]
Couple of comments, rest looks good

**Comment by bowlofeggs** [comment]
On Mon, 2016-12-19 at 01:50 -0800, Pierre-Yves Chibon wrote:
> that got me confused a little, still kinda do, but somehow seems to
> work
> (misses the docstring btw)

flake8 gets upset if lambdas are assigned to a variable since that
defeats the purpose of lambdas, so this commit is adjusting them to be
normal functions since that's what they really are.

I'll add the docstring.

**Comment by bowlofeggs** [comment]
On Mon, 2016-12-19 at 01:51 -0800, Pierre-Yves Chibon wrote:
> Should this method be entirely dropped then?

It does get used, and I filed an issue to get the flashing working:

https://github.com/fedora-infra/bodhi/issues/1165

**Comment by pypingou** [comment]
> flake8 gets upset if lambdas are assigned to a variable since that defeats the purpose of lambdas, so this commit is adjusting them to be normal functions since that's what they really are.

No, I got the move from lambda to a method (I don't really like lambda either) but I really meant the construct used here, if I had been writing this, I would have used an ``return foo if foo else bar`` or so. The current construct looks a little magical to me (but it seems to work, so cool :))

**Comment by bowlofeggs** [comment]
@pypingou ah yeah, I agree that it is strange (I didn't write it) but yeah, it does seem to work. I guess ```True and "bowlofeggs"``` resolves to ```"bowlofeggs"```… strange!

**Pull request merged**